packages/modul-components/src/components/photo-editor/image-displayer/image-displayer.html

```diff
@@ -0,0 +1,8 @@
+<div class="m-image-selector">
+    <m-avatar :size="avatarSize">
+        <img v-if="urlPhoto"
+             class="m-image-selector__avatar"
+             :src="urlPhoto"
+             alt="" />
```

Comment:
attribut vide ?

Reply:
J'ai une erreur de lintage si je n'en mets pas et à mon avis il s'agit d'une image décorative donc le alt devrait être vide si tel est le cas. (https://www.w3.org/WAI/tutorials/images/decorative/). Qu'en penses-tu?

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.

> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.